server/store/mongo_store.go

```diff
@@ -43,3 +40,40 @@ func (s *MongoStore) CreateOrder(order *model.Order) error {
 	_, err := collection.InsertOne(context.Background(), order)
 	return err
 }
+
+func (s *MongoStore) ListOrders(status int) ([]*model.Order, error) {
+	collection := s.client.Database(database).Collection(orderCollection)
+	filter := bson.D{{"status", status}}
+	rs, err := collection.Find(context.Background(), filter)
+	if err != nil {
+		return nil, err
```

Comment:
Should we log it before returning?

Reply:
Yes. We should. This is on my todo list. I am exploring how to collect log for the web app. Currently it is running in container.